Dockerfile

```diff
@@ -17,29 +17,22 @@ ENV IMAGETAG=$tag
 RUN echo " ---------------------------------- "
 RUN echo "env variable IMAGETAG is ${IMAGETAG}"
 
+# create default scientific Python environment
 
-
-
-    # create default scientific Python environment
-
-RUN conda config --add channels pyviz/label/dev
-RUN conda config --add channels bokeh/label/dev
-RUN conda config --add channels intake
 RUN conda config --add channels bioconda
 RUN conda config --add channels conda-forge
-#RUN conda update --yes conda
+RUN conda config --set channel_priority strict
```

Comment:
This is also an important change, ensures packages are pulled from conda-forge instead of defaults wherever possible, which results in a more consistent environment.